server/application-server/src/test/resources/application-test.yml

```
@@ -0,0 +1,43 @@
+# Minimal test configuration for Spring Boot tests
+spring:
+  # Database configuration for tests (overridden by Testcontainers)
+  datasource:
+    # These values will be overridden by @DynamicPropertySource in test classes
+    url: jdbc:postgresql://localhost:5432/test
+    driver-class-name: org.postgresql.Driver
+    username: test
+    password: test
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)